docs/pages/setup/reference/metrics.mdx

```diff
@@ -44,7 +51,6 @@ Now you can see the monitoring information by visiting several endpoints:
 | `backend_read_seconds` | histogram | cache | Latency for read operations. |
 | `backend_write_requests_total` | counter | cache | Number of write requests to the backend. |
 | `backend_write_seconds` | histogram | cache | Latency for backend write operations. |
-| `certificate_mismatch_total` | counter | Teleport Proxy | Number of times there was a certificate mismatch. |
```

Comment:
Why was this removed?

In the source, the description is `counts login failures due to certificate mismatch`

Reply:
I moved this to the H2 section related to the Proxy. Looking at the source, though, this is [incremented](https://github.com/gravitational/teleport/blob/c0205a737108c0a43b15f0781bf234992952b9e9/lib/srv/authhandlers.go#L288) in `*AuthHandlers.UserKeyAuth`, which seems to be used when initializing SSH servers in both Nodes and the Proxy. I've suggested adding this to the "All Teleport Instances" section.